lib/process.js

```diff
@@ -139,7 +139,8 @@ module.exports = function (config, url, cb) {
 		port: parsedUrl.port || (parsedUrl.protocol === 'https:' ? 443 : 80),
 		path: parsedUrl.pathname || '/',
 		method: 'GET',
-		auth: config.http && config.http.auth
+		auth: config.http && config.http.auth,
+		headers: config.http.headers
```

Comment:
This line should be `headers: config.http && config.http.headers`

Reply:
Not sure to follow you here? This would set httpOptions.headers to a boolean value instead of the wanted value in the config.json ?

Reply:
> This would set httpOptions.headers to a boolean value 

Nope. if `config.http` is undefined, it will return `undefined`. Same for null or empty string. `&&` would short-circuit.

If `config.http` is not "falsy" it will return what ever is in `config.http.auth`. Only when needed will the final result be converted into a boolean (with `!` or `!!` or `if ()` or `while ()` or...